src/config/index.ts

```diff
@@ -421,4 +337,130 @@ config = merge(
   { arrayMerge: overwriteMerge }
 )
 
+// load local config files
+if (process.env.LOAD_JSON_CONFIGS) {
+  const configs = process.env.LOAD_JSON_CONFIGS.split(',')
+  for (let i = 0; i < configs.length; i++) {
+    configs[i] = configs[i].trim()
+  }
+
+  for (let i = 0; i < configs.length; i++) {
```

Comment:
any particular reason for allowing configs to be an array? shouldn't it be a single file?
Multiple configs means there is a chance we overwrite config value from previous file. I guess having single file will avoid confusion and errors?
@urnotsam your views?

Reply:
It was my understanding we were going to allow for the use of multiple config files at once which would require us to use an array.

Reply:
correct, andrew specifically asked for multiple config files. the idea being that if we wanted to apply the debug-10-nodes config as well as a new config file for EVM testing(for example), we could do so easily